Makefile

```diff
@@ -20,10 +20,11 @@ PLATFORMS ?= linux/amd64,linux/arm64
 DOCKER_BUILDX_CMD ?= docker buildx
 IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
 IMAGE_BUILD_EXTRA_OPTS ?=
-IMAGE_REGISTRY ?= gcr.io/k8s-staging-jobset
-IMAGE_NAME := jobset
+STAGING_IMAGE_REGISTRY := us-central1-docker.pkg.dev/k8s-staging-images
+IMAGE_REGISTRY ?= $(STAGING_IMAGE_REGISTRY)/jobset
```

Comment:
This is not where our staging repo is, it is under `gcr.io/k8s-staging-jobset` and didn't move yet to artifact registry. Maybe we should do that?

Reply:
Yea, let's do that. 

I think GCR is being shut down next month.

Reply:
How do we do that?

Reply:
How do we do this? We should do that for LWS as well.

Reply:
lets ask on slack then :)

Reply:
I think we can say we figured this out.